tasks/create_vg.yml

```diff
@@ -9,6 +9,19 @@
     - vg.create is defined
     - vg.create|bool
 
+### workaround: auto pvresize waiting for upgrade to new module supporting integrated pvresize
+###             ref: https://docs.ansible.com/ansible/3/collections/community/general/lvg_module.html
+- name: create_vg | pvresize to max (implemented from ansible 2.10 and above)
```

Comment:
Probably don't need this in the task name: ` (implemented from ansible 2.10 and above)`

Reply:
You're right... commited changed task comment (48e3be3c5c8432d59189ecd90ef8fa66b1fa851c)